src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import lotto.view.ResultView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoMachine {
+    private static final int PRICE = 1000;
```

Comment:
돈과 같은 단위는 언더바를 통해 좀 더 가독성을 높여볼 수 있을 것 같아요 😄 
```suggestion
    private static final int PRICE = 1_000;
```

Reply:
수정하였습니다.